### DISCUSSION TITLE
Top level help statements

### DISCUSSION THREAD

**Discussion by xenador1** [nat-n/poethepoet#147] [Q&A]
I'm relatively new to python development, poetry, and poethepoet so I hope I haven't missed something obvious... but is there a way to have: `poetry run poe --help` include the description value (or any other element) as a top-level help message?
Specifically, I have a `./pyproject.toml` file defined and I've been able to see task help messages but would love an overarching help message for what I view as a single CLI tool.
EG:
```
[tool.poetry]
// ...
description = "Can I show this when the help is shown?"
// ...

[tool.poe.tasks.best-task]
help = "This shows"
// ...
```

If it helps I'm using poetry: 1.4.2
poethepoet version: 0.17.1
and my target platform is a linux system.

**Comment by nat-n** [comment]
Hi @xenador1, 

So you want to have the name/description of your project included in the the page of help output from poe along with the list of tasks etc?
This is not currently supported. Could you explain a bit more about your use case? perhaps this could motivate adding it as a feature.
